### ISSUE TITLE
Graph links on kghub.org to kghub.io are broken

### ISSUE THREAD

**Issue by caufieldjh** [Knowledge-Graph-Hub/knowledge-graph-hub-support#14]
### Describe the bug

Links in the KG-Hub Projects table on https://kghub.org/ leading to kghub.io are broken.

### To Reproduce

On kghub.org, click this link: https://kghub.io/kg-obo/

Currently, it leads to an error page:
```xml
<Error>
  <Code>NoSuchKey</Code>
  <Message>The specified key does not exist.</Message>
  <Key>kg-obo/</Key>
  <RequestId>R17C16AMDFXJD20F</RequestId>
  <HostId>9f4KtlTnd5QqODqiBlZ3DFRwzkqYyZgFvVXwljyzLEiqlRy7NV4O5xuLsrZePkAl8wpSokdQ6ec=</HostId>
</Error>
```

## Expected behavior

The link above should go to https://kghub.io/kg-obo/index.html.

**Comment by kltm** [comment]
Is this still open?

**Comment by caufieldjh** [comment]
Fixed as per #18
